students/arun_nalla/session03/Lesson3_Mailroom.py

```diff
@@ -0,0 +1,62 @@
+#!usr/bin/en python
+donors_list = [('Aron 123', [10,5]), ('Becky', [3,30]), ('Charlie',[10,26]),('Thomas',[22,9,4,5]),('Nancy',[1,2,3])]
+donors_name = [x for x,y in donors_list]
+donors_number = {x: len(y) for x,y in donors_list}
+donors_total = {x:sum(y) for x,y in donors_list}
```

Comment:
Even dict comprehensions! But I'm not sure if these will be useful, because these will have to be recomputed whenever a new donation or a new donor is added?